akamai/cache-client.go

```diff
@@ -275,3 +273,18 @@ func (cpc *CachePurgeClient) Purge(urls []string) error {
 	cpc.stats.Inc("SuccessfulPurges", 1)
 	return nil
 }
+
+var akamaiBatchSize = 100
+
+// Purge attempts to send a purge request to the Akamai CCU API cpc.retries number
+//  of times before giving up and returning ErrAllRetriesFailed
+func (cpc *CachePurgeClient) Purge(urls []string) error {
+	for i := 0; i < len(urls); {
+		err := cpc.purgeBatch(urls[i:int(math.Min(float64(i+akamaiBatchSize), float64(len(urls))))])
```

Comment:
Could you pull out this logic for the high end of the slice into its own variable? Also, casting to float64 and back to int just to make use of `math.Min` is not Go style - instead, I think it's simpler to say something like

```go
max := i+akamaiBatchSize
if max > len(urls) {
  max = len(urls)
}
```